python/ray/autoscaler/aws/cloudwatch/prometheus_ray_up.sh

```diff
@@ -0,0 +1,35 @@
+###########################################################################################################
+#This bash script is meant to support prometheus metrics collection via cloudwatch agent.
+#You can view the ray metrics being collected through AWS CloudWatch console.
+###########################################################################################################
+
+read -r -p $'Usage: prometheus_ray_up.sh [OPTIONS] CLUSTER_CONFIG_FILE:\n' RAY_UP_OPTIONS
+AUTOSCALER_CONFIG_FILE=$(sed 's/.* //' <<< "$RAY_UP_OPTIONS")
+CLUSTER_NAME=$(grep 'cluster_name' $AUTOSCALER_CONFIG_FILE| cut -d ':' -f 2 | xargs)
+
+# ray up to start a new cluster
+ray up $RAY_UP_OPTIONS
```

Comment:
This script will primarily be invoked in a non-interactive environment, so we should get rid of the prompt here and instead replace it with a simple command line argument parser/forwarder like:

```sh
FLAGS=""
AUTOSCALER_CONFIG_FILE=""
PARAMS=""
USAGE_INSTRUCTIONS='Usage: prometheus_ray_up.sh [OPTIONS] CLUSTER_CONFIG_FILE'


while (( "$#" )); do
  case "$1" in
    -*|--*=) # flags
      FLAGS="$FLAGS $1"
      shift
      ;;
    *) # positional arguments
      if [ -z "$AUTOSCALER_CONFIG_FILE" ]
      then
        AUTOSCALER_CONFIG_FILE="$1"
      else
        echo "$USAGE_INSTRUCTIONS"
        exit 1
      fi
      shift
      ;;
  esac
done

if [ -z "$AUTOSCALER_CONFIG_FILE" ]
then
  echo "$USAGE_INSTRUCTIONS"
  exit 1
fi

CLUSTER_NAME=$(grep 'cluster_name' $AUTOSCALER_CONFIG_FILE| cut -d ':' -f 2 | xargs)

# ray up to start a new cluster
ray up $FLAGS $AUTOSCALER_CONFIG_FILE
```